### PR TITLE
Add jAlbum.app v13

### PR DESCRIPTION
**Pull request by gfilicetti**
Initial creation of jAlbum.app
First attempt at jAlbum cask
Passed install / uninstall test
Passed audit
Passed style check

### CODE REVIEW

File: Casks/jalbum.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+  # amazonaws.com/gateblu was verified as official when first introduced to the cask
+  url "http://download.jalbum.net/download/#{version}/MacOSX/jAlbum.dmg"
+  name 'jAlbum'
+  homepage 'https://jalbum.net'
+  license :freemium
+
+  app 'jAlbum.app'
+
+  uninstall delete:  '/Applications/jAlbum.app'
```
Comment (vitorgalvao) [inline Casks/jalbum.rb]: Remove this line. Removal is done automatically, and you can’t assume people will install to this directory.

File: Casks/jalbum.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+cask 'jalbum' do
+  version '13'
+  sha256 :no_check
+
+  # amazonaws.com/gateblu was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/jalbum.rb]: This comment is wrong and needs to be removed.

File: Casks/jalbum.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+cask 'jalbum' do
+  version '13'
+  sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/jalbum.rb]: `sha256` also needs to be set. You can find what it is by running `shasum -a 256` on the downloaded file.

Comment (gfilicetti) [inline Casks/jalbum.rb]: Unfortunately, even though the download URL is versioned, it is versioned via the major release, ie: 13. The actual file you get can be any minor version of 13 as the app adheres to continuous release practices and therefore would have a different checksum each time.

File: Casks/jalbum.rb (hunk 2 above)
Comment (gfilicetti) [inline Casks/jalbum.rb]: Ugh... sorry about that.

File: Casks/jalbum.rb (hunk 3 above)
Comment (vitorgalvao) [inline Casks/jalbum.rb]: That may or may not be a problem, depending on how often it‘s updated. It’s better to err on the other side. Set the `sha256` and if it becomes unmanageable in the future, we’ll remove it.

**Comment by vitorgalvao** [comment]
Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by gfilicetti** [comment]
I believe the squash is ok?

**Comment by vitorgalvao** [comment]
Nope, the goal is to end up with one commit. There are currently three. Don’t worry if it doesn’t go well at first, it’s an obtuse process that trips up most people at first.

**Comment by gfilicetti** [comment]
Alright vitor... that should do it. I was messing up on the rebase, leaving > 1 pick line. All set now I hope?!

**Comment by vitorgalvao** [comment]
Yep. Thank you.

**Comment by gfilicetti** [comment]
Hey Vitor... one last thing re: a contribution on github... should I tie off or delete my fork now that my (current) submission is done? Or do they just build up and up and up?

**Comment by vitorgalvao** [comment]
Whatever you feel like. They do tend to pile up (at least for me), but if you’re planning on contributing more, keeping the for will make it faster (no new forks will be created for this repo).

Deleting is also not a big issue, though.

My advice: don’t delete for now, and once every few months look at your repos and get rid of the cruft.

**Comment by gfilicetti** [comment]
Awesome, thanks for the help.

As I keep reinstalling my applications on this mac (just wiped and
installed el capitan yesterday).

What do you think? Separate pull requests for each cask?

G

**Comment by vitorgalvao** [comment]
Yes, we require one PR per cask.

**Pull request merged**